src/SubsCache.tests.js

```diff
@@ -279,15 +279,17 @@ describe("SubsCache - subscribe", function() {
 
   it("delete the oldest subscription if the cache has overflown", function() {
     var subsCache = new SubsCache(5, 1);
```

Comment:
as i get it the limit is only relevant for stopped/inactive migrations - only subscriptions with count = 0 will actually be deleted when a new one is created which overflows the limit. So the idea was to make the `subAll` subscription going inactive so that it can be removed.